lib/rules/no-invalid-regexp.js

```diff
@@ -10,7 +10,7 @@
 
 const RegExpValidator = require("regexpp").RegExpValidator;
 const validator = new RegExpValidator();
-const validFlags = /[gimuys]/gu;
+const validFlags = /[gimuysd]/gu;
```

Comment:
```suggestion
const validFlags = /[dgimsuy]/gu;
```
nit, but .flags and .toString sort them this way, so it might make it easier to spot omissions or mistakes?

Reply:
I think these flags are in the order they were added after ES2015.

https://github.com/eslint/eslint/pull/10062/files#diff-0ffe225832710a1def4652b09da3d4b603cdfd0f40bdd212618f51cfc708148fR13

If the ESLint team prefers alphabetical order, I will change the order of the flags in this PR.

Reply:
:+1: makes sense to sort them alphabetically now.